service/src/chain_spec.rs

```diff
@@ -125,6 +125,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 			minimum_validator_count: 4,
 			stakers: initial_authorities.iter().map(|x| (x.0.clone(), x.1.clone(), STASH, StakerStatus::Validator)).collect(),
 			invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
+			.. Default::default()
```

Comment:
Note: that's the `force_era` field of https://github.com/paritytech/substrate/pull/3374

Reply:
I think the intention is that this one is supposed to be `Forcing::ForceNone`, but you should get @gavofyork to confirm